java/com/google/re2j/Machine.java

```diff
@@ -28,47 +28,45 @@
   // research.swtch.com/2008/03/using-uninitialized-memory-for-fun-and.html
   private static class Queue {
 
-    final Thread[] dense; // may contain stale Entries in slots >= size
+    final Thread[] denseThreads; // may contain stale Entries in slots >= size
+    final int[] densePcs;  // may contain stale Entries in slots >= size
```

Comment:
In Go this would be capitalized as densePCs; I can't remember what the Java convention is.